index.php

```diff
@@ -28,260 +29,330 @@
 /**
  * Generates main chess page, which displays lists of games and challenges.
  *
- * @package chess
+ * @package    chess
  * @subpackage index
  */
 
+use Xmf\Request;
+
 /**#@+
  */
-include '../../mainfile.php';
-$xoopsOption['template_main'] = 'chess_games.html';
-include_once XOOPS_ROOT_PATH . '/header.php';
+require dirname(dirname(__DIR__)) . '/mainfile.php';
```

Comment:
`header.php` loads `mainfile.php`
```suggestion
require __DIR__ . '/header.php';
```